package.json

```diff
@@ -18,7 +18,8 @@
     "solium-autofix": "solium -d contracts/ --fix",
     "predeploy-contracts": "mkdir -p ./build/contracts && find ./artifacts/deployed/ -name \\*.json -exec cp {} ./build/contracts/ \\;",
     "deploy-contracts": "cross-env ACCOUNT_PASSWORD=privatenet123 NODE_ENV=test truffle migrate --network ${NETWORK}",
-    "ci": "npm-run-all test lint"
+    "check": "npm-run-all test lint",
+    "check-ci": "npm-run-all test-contracts lint"
```

Comment:
Why do we use test-contracts in ci instead of test?

Reply:
@christiansaiki `test` brings own ganache service up/down and CI server starts it's own ganache container, so we do not need to duplicate. Otherwise both commands do the same job.